### PR TITLE
python-eve framework

### PR DESCRIPTION
**Pull request by Denommus**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -4522,6 +4537,39 @@ in modules // {
...
+      jinja2
+      markupsafe
+      pymongo_2_9_1
+      simplejson
+      werkzeug
+
+    ];
+
+    # tests call a running mongodb instance
+    doCheck = false;
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: Is it (with a reasonable amount of effort) possible to run the tests that do not depend on a mongodb instance?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -2492,6 +2492,21 @@ in modules // {
...
+    name = "Cerberus-${version}";
+    version = "0.9.2";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/C/Cerberus/${name}.tar.gz";
+      sha256 = "1km7hvns1snrmcwz58bssi4wv3gwd34zm1z1hwjylmpqrfrcf8mi";
+    };
+
+    meta = {
+      homepage = http://python-cerberus.org/;
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: Nitpick: You don't need quotes around URLs.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -15920,6 +16002,25 @@ in modules // {
...
+    name = "pymongo-2.9.1";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "http://pypi.python.org/packages/source/p/pymongo/${name}.tar.gz";
+      sha256 = "1nrr1fxyrlxd69bgxl7bvaj2j4z7v3zaciij5sbhxg0vqiz6ny50";
+    };
+
+    # Tests call a running mongodb instance
+    doCheck = false;
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: See comment about tests from above.

**Comment by pSub** [comment]
Otherwise it looks good!

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (Denommus) [inline pkgs/top-level/python-packages.nix]: I would have to investigate further (and I promise to do it when I have the time).

**Comment by FRidh** [comment]
ping

**Comment by pSub** [comment]
Sorry for the delay. The refactoring of the tests can be done later.

**Pull request merged**